src/Public/js/sis.selector.js

```diff
@@ -23,15 +25,31 @@ var Selector = function () {
       }
 
       if (locked) {
-        myradio.createDialog("Error", "Could not change studio.<br>Studio selector is currently locked out.");
+        myradio.createDialog("Selector Error", "Could not change studio.<br>Studio Selector is currently locked out.");
         return;
       }
+
+      if (studio == 8) {
```

Comment:
why only a confirm for offair? seems like the sort of thing you should confirm regardless - we've definitely seen instances of people accidentally switching to jukebox before

Reply:
Fair enough, will do :)

Reply:
Done